src/test/scala/uk/gov/hmrc/api/service/DisaSubmissionService.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.api.service
+
+import play.api.libs.ws.StandaloneWSResponse
+import uk.gov.hmrc.api.client.HttpClient
+import uk.gov.hmrc.api.conf.TestEnvironment
+import uk.gov.hmrc.api.utils.SubPathGenerator
+
+import scala.concurrent.Await
+import scala.concurrent.duration.*
+
+class DisaSubmissionService extends HttpClient {
+  val host: String    = TestEnvironment.url("disa-submission-backend")
+  val subPath: String = SubPathGenerator.generateSubpath()
+
+  def post(
+    payloadFileName: String
+  ): StandaloneWSResponse =
+    Await.result(
+      post(
+        host + subPath,
+        payloadFileName,
+        ("Content-Type", "application/x-ndjson")
+      ),
+      10.seconds
+    )
```

Comment:
This looks like it's trying to be a connector - lets call it that. If there is a testing guideline that suggests to do it this way I'd like to see it.